src/test/regress/sql/local_shard_utility_command_execution.sql

```diff
@@ -0,0 +1,304 @@
+-- This tests file includes tests for local execution of utility commands.
+-- For now, this file includes tests only for local execution of
+-- `TRUNCATE/DROP/DDL` commands for all kinds of distributed tables from
+-- the coordinator node having regular distributed tables' shards
+-- (shouldHaveShards = on) and having reference table placements in it.
+
+\set VERBOSITY terse
+
+SET citus.next_shard_id TO 1500000;
+SET citus.shard_replication_factor TO 1;
+SET citus.enable_local_execution TO ON;
+SET citus.shard_COUNT TO 32;
+SET citus.log_local_commands TO ON;
+
+CREATE SCHEMA local_commands_test_schema;
+SET search_path TO local_commands_test_schema;
+
+-- let coordinator have distributed table shards/placements
+SELECT master_add_node('localhost', :master_port, groupId => 0);
+SELECT master_set_node_property('localhost', :master_port, 'shouldhaveshards', true);
+
+-----------------------------------------
+------ local execution of TRUNCATE ------
+-----------------------------------------
+
+CREATE TABLE ref_table (a int primary key);
+SELECT create_reference_table('ref_table');
+
+CREATE TABLE dist_table(a int);
+SELECT create_distributed_table('dist_table', 'a');
```

Comment:
can you add one more colocated distributed table, which is partitioned, and 1-2 partitions along with that? At the end of the, please add some simple sanity checks that all the logic works fine for partitioned tables as well?